### PR TITLE
[TAAS-18] A maximally viable countries API.

### PR DESCRIPTION
**Pull request by pjf**
Example record:

```
{
    "admin_level": "0", 
    "geolocation": {
        "lat": "32.57533772", 
        "lon": "54.2973284"
    }, 
    "hrinfo_id": "285", 
    "id": "105", 
    "iso2": "IR", 
    "iso3": "IRN", 
    "label": {
        "default": "Iran (Islamic Republic of)", 
        "humanitarianresponse": "Iran, Islamic Republic of", 
        "m49": "Iran (Islamic Republic of)", 
        "reliefweb-api": "Iran (Islamic Republic of)", 
        "reliefweb-short": "Iran"
    }, 
    "m49": "364", 
    "reliefweb_id": "121", 
    "x-alpha-2": "IR", 
    "x-alpha-3": "IRN"
}
```

This includes PR #18 (the minimal API).

Discussions to be linked to on JIRA when I open a ticket there shortly.

### CODE REVIEW

File: taas/config.yml [hunk 1]
```diff
@@ -54,3 +54,88 @@ sources:
...
+                    field: ISO 3166-3:2013 Alpha 3-Codes
+                    optional: True
+
+                iso2:
+                    field: ISO 3166-2:2013 Alpha 2-Codes
+                    optional: True
+
+                # Unofficial but generally used fields. These
+                # default to the ISO code where it exists.
+                x-alpha-2:
```
Comment (emmajane) [inline taas/config.yml]: `These default` sounds like an automated defaulting. Is that the case? If not, perhaps this comment could be: "Where there is an ISO code, these fields will contain the same value as the ISO code."

File: taas/config.yml [hunk 2]
```diff
@@ -54,3 +54,88 @@ sources:
...
+                    field:
+                        - x Alpha2 codes
+                        - ISO 3166-2:2013 Alpha 2-Codes
```
Comment (emmajane) [inline taas/config.yml]: Yes, looks like an automated "failover" if blank...?

File: taas/config.yml [hunk 3]
```diff
@@ -54,3 +54,88 @@ sources:
...
+                        - x Alpha3 codes
+                        - ISO 3166-3:2013 Alpha 3-Codes
+                    optional: True
+
+                # The old HRinfo ID, in case people want to match.
+                hrinfo_id:
+                    field: HRinfo ID
+                    optional: True
+
+                # Ditto ReliefWeb
```
Comment (emmajane) [inline taas/config.yml]: Replace "Ditto" with the full explanation: `The old ReliefWeb ID, in case people want to match.`

**Comment by pjf** [comment]
@emmajane : Comments updated in 2099c36! Merge imminent. :)

**Pull request merged**